department-of-veterans-affairs/caseflow
# BillyS/APPEALS-8972: Ready for review updates

Resolves [APPEALS-8972](https://vajira.max.gov/browse/APPEALS-8972)

### Description
Adjustments for the Ready for review modal surfacing from the dogfooding session.  As a CSP user, I need the verbiage to display correctly on the Ready for review modal as well as success banner.

### Acceptance Criteria
   - [ ] Ready for review title has a lowercase "r" in "review"
   - [ ] Subsequent confirmation banner has a lowercase "r" in "review"

### Testing Plan
1. Open Caseflow App
2. Log in as Intake User
![image](https://user-images.githubusercontent.com/98414141/185998025-087f720a-6ce8-4726-8d3c-e371e814f805.png)
3. Select "Decision Review Request: Board Appeal (Notice of Disagreement) — VA Form 10182".
4. Enter the following VA file number: `500000001`
5. Follow the 10182 input flow until you get to the Add/Remove Issues page.
6. Click the `Add Issue` button:
<img width="535" alt="add_issue_button" src="https://user-images.githubusercontent.com/99351305/177413069-f2a82b58-b04a-4014-9c39-1807b9b8b44d.PNG">

7. Select "Veteran Health Administration" from the "Benefit type" dropdown.
8. Select one of the Caregiver issue types. Enter any description for the issue you'd like.
9. Click the "Add this Issue" button.
10. Click the "Submit appeal" button.
11. Select "Begin next intake" button.
12. Follow steps 3 through 10 to do another intake as a VHA Camo user making these modifications:
      a. in step 4 use VA file number `500000002`
      b. choose option OTHER than Caregiver issue type
13. Follow steps 3 through 10 to do another intake as an Education user making these modifications:
      a. in step 4 use VA file number `500000003`
      b. in step 7 choose Education and yes for Pre-Docket
      c. choose any option available
14. Follow step 13 to do another intake as an Education user making these modifications:
      a. in step 4 use VA file number `500000004`

15. Switch user to Caregiver user
![image](https://user-images.githubusercontent.com/98414141/182457704-7b5dbe54-8745-489b-bb46-c5964ad0b2a5.png)
16. Select Your Queue
![image](https://user-images.githubusercontent.com/98414141/182458385-e6e710b3-f3b8-41ed-b8f6-e090de6e481d.png)

17. Switch to the CSP organization's queue:
<img width="852" alt="switch-to-csp-queue" src="https://user-images.githubusercontent.com/99351305/181598516-51ff9792-ba56-4bcd-b9fd-e8aa3776954e.PNG">

18. Find the case you set up on the "Unassigned Tab" and select it to open the case details page
![image](https://user-images.githubusercontent.com/98414141/189340484-df413eac-27d8-4c3d-b8ae-9a74deb61137.png)

19. Go to the dropdown, find the task action "Documents Ready for Board Intake review" and select it
![image](https://user-images.githubusercontent.com/98414141/189340735-9e8a8b3f-f69c-48c4-b33c-4401b83a90b6.png)

20. When the modal appears, please confirm that the title says "Ready for review" and not "Ready for Review"
![image](https://user-images.githubusercontent.com/98414141/189340982-3d4d3865-f0d3-44a9-8cb3-5d94f4f0ddc1.png)

21. Select VBMS radio button then click Send.  This should bring you to success banner.  Confirm that the last word in the banner is "review" and not "Review"
![image](https://user-images.githubusercontent.com/98414141/189341333-9fb3b7bc-3e35-4f68-9a19-5a2fdcc7c194.png)

22. Repeat steps 15 - 21 for EMO user accessing appeal with VA file Number '500000003', Task action in step 19 will be "Ready for Review"
23. Go back to the EMO Queue Unassigned tab and select the other appeal '500000004'
24. Select the task action "Assign to Regional Processing Office"
![image](https://user-images.githubusercontent.com/98414141/189359332-437c4944-7d62-4691-a912-eff9cea3e511.png)

25. In the modal, select "Buffalo RPO" and click "Submit"
26. Repeat steps 15 - 21 for RPO user accessing appeal with VA file Number '500000004', Task action in step 19 will be "Ready for Review"
27. Switch to Camo User Queue "Assigned" tab and select appeal with VA file Number '500000002'
28. Select the task action "Assign to Program Office" and assign to one of the program offices
![image](https://user-images.githubusercontent.com/98414141/189366250-0a325795-4f36-4803-9516-a7f2ee66db80.png)

30. Repeat steps 15 - 21 for VHA PO user accessing appeal with VA file Number '500000002', Task action in step 19 will be "Ready for Review"
